app/transcription.py

```
@@ -152,6 +179,27 @@ def __configure_review_flag(self, needs_review):
+        folder = os.path.join(self.metadata_writer.base_dir, loc)
+        for filepath in glob.glob(os.path.join(folder, "*", "metadata_*.json")):
+            try:
+                with open(filepath) as fh:
+                    data = json.load(fh)
+                if data.get("source_file") == source_file or data.get("media") == source_file:
+                    self.logger.info(
+                        f"Cache hit for '{source_file}' — loading metadata from disk, skipping yt_dlp."
+                    )
+                    return data
+            except Exception:
+                continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)